examples/workspace_access.tf

```diff
@@ -0,0 +1,15 @@
+data "prefect_workspace_role" "developer" {
+	name = "Developer"
+}
+data "prefect_workspace" "prd" {
+	id = "<workspace uuid>"
+}
+resource "prefect_service_account" "bot" {
+	name = "a-cool-bot"
+}
+resource "prefect_workspace_access" "bot_access" {
+	accessor_type = "SERVICE ACCOUNT"
+	accessor_id = prefect_service_account.bot.id
+	workspace_id = data.prefect_workspace.prd.id
+	workspace_role_id = data.prefect_workspace_role.developer.id
+}
```

Comment:
can you introduce terraform fmt as a precommit dep or as a PR linter?

Reply:
ignore i see that you did in https://github.com/PrefectHQ/terraform-provider-prefect/pull/77/files

Reply:
great minds. i was like...something feels like it's missing